README.md

```diff
@@ -3,18 +3,20 @@
 A way for you to track your in-game statistics (overall and per-park) such as:
 
 - Time you've spent in-game
+- Number of rides you've built
+- Number of vehicles you've crashed
```

Comment:
All changes in this file should be reverted and only applied to the other PRs that would add the functionality.